civihr_employee_portal/js/reports.js

```diff
@@ -919,6 +919,67 @@
                         }
 
                         this.$DOM.sections.dataWrapper.fadeIn();
+                        var $DOM = this.$DOM;
+
+                        $.ajax({
```

Comment:
Is it necessary to have this `$.ajax` call in the `ready` handler of the data table DOM object? That is, are you sure you need to create the summary table only when the data table is built and displayed? Can't they be done in parallel, since  it seems you are not overwriting the `data` object returned by the initial ajax call?